src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -952,7 +964,7 @@ def load_der_public_key(self, data):
             else:
                 self._handle_key_loading_error()
 
-    def load_der_parameters(self, data):
+    def load_der_parameters(self, data: bytes) -> _DHParameters:
```

Comment:
Why private?

Reply:
ugh, I fixed this on pem but missed der.